norddrop/src/ffi/mod.rs

```diff
@@ -373,6 +373,41 @@ pub extern "C" fn norddrop_new(
     });
 
     let result = panic::catch_unwind(move || {
+        {
```

Comment:
I would say, let's use it under the feature flag. We pretty much don't want this in the release builds

Reply:
It will invoke `slog` macros which are going to be compiled out. `slqx` will also use the provided severity so it's pretty good. I've added additional guard just in case